src/components/atoms/FileExplorer/FileExplorer.tsx

```diff
@@ -14,57 +15,26 @@ export type FileExplorerOptions = DirectoryOptions | FileOptions;
 
 export type FileExplorerProps = {
   isOpen: boolean;
-  onSelect: (files: FileList) => void;
+  onSelect: (files: string[]) => void;
   onOpen: () => void;
   options?: FileExplorerOptions;
 };
 
 const FileExplorer = (props: FileExplorerProps) => {
   const {isOpen, onSelect, onOpen, options} = props;
-  const fileInputRef = useRef<HTMLInputElement>(null);
 
   useEffect(() => {
     if (isOpen) {
-      fileInputRef && fileInputRef.current?.click();
       onOpen();
+      ipcRenderer.invoke('select-file', options).then(files => {
```

Comment:
Additionally, we can add this to the store too. After my PR is merged we will be able to retrieve state and trigger actions in the main process.

Or we can leave it for a later refactoring session.

Reply:
I can do it in a later refactoring!